### ISSUE TITLE
Yewtube v2.10.4 suggests upgrading to v2.9.2

### ISSUE THREAD

**Issue by LeXofLeviafan** [mps-youtube/yewtube#1258]
After using the `q`/`quit` command, I get the following message printed out:
```
A newer version is available (2.9.2). Use `help new` command to check what's changed.
```
Problem is, the splashscreen says that the installed version is `v2.10.4`. Which means the latest version detection works incorrectly (most likely treating version numbers either as strings or as decimal fractions).

**Comment by iamtalhaasghar** [comment]
That's because the github action for auto-deployment to pypi is broken. You might have installed yewtube from github and thus the wrong message.

**Comment by LeXofLeviafan** [comment]
I installed it using the `pipx install yewtube` command.

**Comment by eclairevoyant** [comment]
Should have been fixed by #1262
